src/components/widget/DatePicker.js

```diff
@@ -87,15 +87,17 @@ class DatePicker extends Component {
 
   render() {
     return (
-      <div tabIndex="-1" onKeyDown={this.handleKeydown}>
+      <div tabIndex="-1" onKeyDown={this.handleKeydown} className="datepicker">
         <Datetime
           closeOnTab={true}
           renderDay={this.renderDay}
           renderInput={this.renderInput}
           onBlur={this.handleBlur}
           onFocus={this.handleFocus}
           {...this.props}
+          open={true}
```

Comment:
Also in edit mode. After doubleclick on the datefield to switch into edit more, the datepicker does not dissappear after selection anymore.
![image](https://user-images.githubusercontent.com/13365687/34861296-1a11c024-f764-11e7-8317-ba4ba1032efe.png)